### DISCUSSION TITLE
model = torch.load(path, map_location=self.device)["model"] KeyError: 'model

### DISCUSSION THREAD

**Discussion by EricHuiK** [open-edge-platform/anomalib#1285] [Q&A]
### What is the motivation for this task?

python  tools/inference/torch_inference.py \
        --weights /disk4t/workspace/code/anomalib-main/results/fastflow/mvtec/bottle/run/weights/lightning/model-v7.ckpt \
        --input   /disk4t/workspace/data/mvtec-ad/bottle/test/contamination/019.png \
        --output  /disk4t/workspace/data/val_bottle \
        --device cuda:0 \
        --task segmentation \
        --visualization_mode full

### Describe the solution you'd like

 model = torch.load(path, map_location=self.device)["model"] KeyError: 'model

**Comment by samet-akcay** [comment]
Hi @EricHuiK, can you please set the `export_mode` to torch? By default, it is for lightning.
https://github.com/openvinotoolkit/anomalib/blob/cdedf05451cc57fb373866595e8ef23a820cf123/src/anomalib/models/fastflow/config.yaml#L70-L71

anomalib will then create the torch model rather than the lightning model.
